src/main.qml

```diff
@@ -19,7 +19,7 @@
 import QtQuick 2.9
 import org.asteroid.controls 1.0
 import Nemo.Configuration 1.0
-import 'qrc:/icons.js' as IconTools
```

Comment:
Why remove the qrc prefix?

Reply:
It works either way because the qrc file qresource prefix is set to "/" but I'll revert it back.